vms/platformvm/state/validator_metadata.go

```diff
@@ -60,7 +80,7 @@ func parseValidatorMetadata(bytes []byte, metadata *validatorMetadata) error {
 		// potential reward and uptime was stored but potential delegatee reward
 		// was not
 		tmp := preDelegateeRewardMetadata{}
-		if _, err := txs.Codec.Unmarshal(bytes, &tmp); err != nil {
+		if _, err := validatorMetadataCodec.Unmarshal(bytes, &tmp); err != nil {
```

Comment:
Do we keep backward compatibility for these? I.e what happens to the bytes we already stored?